shifthelper/checks/__init__.py

```diff
@@ -115,15 +115,27 @@ def __init__(self,
     def run(self):
         while not self.stop_event.is_set():
             if len(self.queue) > 0:
```

Comment:
If I read this correctly this statement is unnecessary. The while condition will catch it.